testsuite/tests/typing-missing-cmi/test.compilers.reference

```diff
@@ -3,5 +3,7 @@ File "main.ml", line 1, characters 14-17:
                   ^^^
 Error: This expression has type "M.b" but an expression was expected of type
          "M.a"
-       "M.b" is abstract because no corresponding cmi file was found in path.
-       "M.a" is abstract because no corresponding cmi file was found in path.
+       Type "M.b" is abstract because no corresponding cmi file
+       was found in path.
+       Type "M.a" is abstract because no corresponding cmi file
+       was found in path.
```

Comment:
The line breaks here read like a regression caused by the use of `@ ` instead of ` ` in the error message. Why the change?

Reply:
Because the addition of `Type` at the beginning makes the line potentially too long for a standard terminal.
If you think there is no use for line breaks, we can also go back to ` `.